config/engines.yml

```diff
@@ -141,6 +141,14 @@ phpmd:
     - "**.php"
     - "**.module"
     - "**.inc"
+radon:
+  image: codeclimate/codeclimate-radon
+  description: Python tool used to compute Cyclomatic Complexity.
+  community: true
+  enable_regexps:
```

Comment:
Related to earlier discussion around `upgrade_languages`, it feels weird to me that we include `enable_regexps` for `community` engines that we'll never auto-enable. Is the thinking that eventually we'll remove the community restriction for auto-enable?